tests/lint/unittest_expand_modules.py

```diff
@@ -102,23 +102,26 @@ class Checker(BaseChecker):
     @pytest.mark.parametrize(
         "files_or_modules,expected",
         [
-            ([__file__], [this_file]),
+            ([__file__], {this_file["path"]: this_file}),
             (
                 [str(Path(__file__).parent)],
-                [
-                    init_of_package,
-                    test_caching,
-                    test_pylinter,
-                    test_utils,
-                    this_file_from_init,
-                    unittest_lint,
-                ],
+                {
+                    module["path"]: module  # pylint: disable=unsubscriptable-object
```

Comment:
I need to figure out why Pylint thinks a dictionary is an unsubscriptable object. Feels like a Pylint bug.

Reply:
Yes, probably a false positive. Disabling is ok.